photongo/Readme.md

```diff
@@ -122,356 +154,484 @@ enum Role {
+  photon.Comment.Title.Contains("my title"),
+  photon.Post.User.Email.Equals(email),
+).Exec(ctx)
 ```
 
-#### Find all comments belonging to a post of a user
+### Writing Data
 
-```go
-user := client.User.As(user.Where().Email("alice@prisma.io"))
-post := user.Post.As(post.Where().TitleContains("my title"))
-comments, err := post.Comments.FindMany()
-```
+The following methods describe how to write data in the database. These are all mutations on
+a Prisma level. You can create, update and delete data, which maps to insert, update, and
+delete SQL operations respectively.
 
-## Writing Data
+You can optionally connect other nodes or even create new related nodes.
 
-The write examples follow closely with the demo here: https://www.prisma.io/docs/prisma-client/basic-data-access/writing-data-GO-go08/
+#### Create
 
-### Create
+You can create objects which maps to SQL inserts.
 
-#### Create a User
+##### Create a User
 
 ```go
-usr, err := client.User.Create(
-  user.New().Email("alice@prisma.io").Name("Alice"),
-)
+// User.Create has fixed and required arguments for required fields
+user, err := client.User.Create(
+  photon.User.ID.Set("abc43"),
+  photon.User.Email.Set("alice@prisma.io"),
+  photon.User.Age.Set(37),
+).Exec(ctx)
 ```
 
-#### Create a new post and set alice@prisma.io as the author
+##### Create a new post and set alice@prisma.io as the author
 
 ```go
-pst, err := client.Post.Create(
-  post.New().Title("Join us for GraphQL").ConnectAuthor(
-    user.Connect().Email("alice@prisma.io")
-  ),
-)
+post, err := client.Post.Create(
+  photon.User.ID.Set("abc43"),
+  photon.User.Title.Set("Prisma"),
+).ConnectAuthor(
+  photon.User.Email.Equals("alice@prisma.io"),
+).Exec(ctx)
 ```
 
-#### Create a new user with two new posts
+##### Create a new user with two new posts
 
 ```go
-usr, err := client.User.Create(
-  user.New().Email("bob@prisma.io").Name("Bob").CreatePosts(
-    post.New().Title("Follow @prisma on Twitter"),
-    post.New().Title("Join us for GraphQL"),
-  ),
-)
+user, err := client.User.Create(
+  photon.User.Id.Set("abc345"),
+  photon.User.Name.Set("John"),
+  photon.User.Email.Set("john@example.com"),
+  photon.User.Age.Set(50),
+).CreatePost(
+  photon.Post.ID.Set("a"),
+  photon.Post.Title.Set("Follow @prisma on Twitter"),
+).CreatePost(
+  photon.Post.ID.Set("b"),
+  photon.Post.Title.Set("Join us for GraphQL"),
+).Exec(ctx)
 ```
 
-#### Create 1 user, 2 posts and connect an existing post
+##### Create 1 user, a post and connect an existing post
 
 ```go
-usr, err := client.User.Create(
-  user.New().Email("bob@prisma.io").Name("Bob").
-    CreatePosts(
-      post.New().Title("Follow @prisma on Twitter"),
-      post.New().Title("Join us for GraphQL"),
-    ).
-    ConnectPosts(
-      post.Connect().ID(10),
-    ),
-)
+user, err := client.User.Create(
+  photon.User.ID.Set("7d42"),
+  photon.User.Email.Set("bob@prisma.io"),
+  photon.User.Name.Set("Bob"),
+).CreatePost(
```

Comment:
The above example creates multiple posts. How would I create multiple posts in your proposal?

Reply:
Good point, that's currently missing. I'll add it soon